README.md

```diff
@@ -1,2 +1,44 @@
 # model-config-inputs
-Repo for remote modification of model configuration inputs and storage of their metadata
+
+## Overview
+
+This repo is used for remote modification of model configuration inputs and storage of their metadata. 
+
+Since model configuration inputs are so large, we can't store them directly on git. But we can store metadata about them, which gives a view into when and how configuration inputs have changed. This metadata can also be consumed by other projects. 
+
+## Usage
+
+This repository is not committed to directly - think of it more as a representation of what released model configuration inputs are currently on Gadi. 
+
+But, we do have a workflow for adding/updating the actual model configuration inputs on Gadi here: https://github.com/ACCESS-NRI/model-config-inputs/actions/workflows/remote-copy.yml. This is used by regular users to move configurations from shared spaces into the locked-down Release area of `vk83` on Gadi, or the less restrictive Prerelease area of `vk83`. 
+### Using the Workflow
+
+The workflow has the following inputs:
+
+| Description | Type | Workflow Input Name | Required | Default | Example | Notes |
+| ----------- | ---- | ------------------- | -------- | ------- | ------- | ----- |
+| The Github Environment for the given remote | Choice of `string`s | `remote-environment` | `true` | `Gadi Release` | `Gadi Prerelease`, `Setonix Release` | This input determines if the model config input(s) are going to the Release or Prerelease area of the HPC, and sets users to sign-off on the move accordingly |
+| Note describing the reason for the change | `string` | `note` | `false` | N/A | `Moving this input in for the 2025.02.000 release of MODEL` | This input is used to justify the reason for the movement of config inputs, so users who sign-off on the change know why it was made |
+| Remote absolute path to configuration input source file or directory | `string` (path) | `source` | `true` | N/A | `/scratch/tm70/inputs/access-om2/2025.12.000`, `/scratch/tm70/inputs/access-om2/2025.12.000/some.nc` | This path to a directory or file to be copied must be readable by the service user |
+| Remote absolute path to configuration input destination directory | `string` (path) | `target` | `true` | N/A | `/g/data/vk83/configurations/inputs/access-om2/2025.12.000` (for Release area), `/g/data/vk83/prerelease/configurations/inputs/access-om2/2025.12.000` (for Prerelease area) | The destination directory for the copied file/folder. Must be a directory, and mustn't have the same `basename` as the source |
```

Comment:
```suggestion
| Remote absolute path to configuration input destination directory | `string` (path) | `target` | `true` | N/A | `/g/data/vk83/configurations/inputs/access-om2/` (for Release area), `/g/data/vk83/prerelease/configurations/inputs/access-om2/` (for Prerelease area) | The destination directory for the copied file/folder. Must be a directory, and mustn't have the same `basename` as the source |
```
Given the examples above the trailing directory should be omitted for the directory case. I know it is explained below, so maybe it would be better to have the two examples match the input examples, and just show release?